.github/workflows/ci.yml

```diff
@@ -14,6 +14,12 @@ jobs:
     steps:
       - name: Checkout cedar-java
         uses: actions/checkout@v4
+      - name: Checkout cedar
+        uses: actions/checkout@v4
+        with:
+          repository: cedar-policy/cedar
+          ref: main
+          path: /home/runner/work/cedar-java/cedar-java/cedar
```

Comment:
Is the hardcoded path ok here? Is it guaranteed to always be the same in CI?

Reply:
I doubt it changes often, so even if it eventually does I prefer this to keeping the config script around.

Reply:
As an alternative, a relative path could also work in this context. The full path should not change often, if at all, unless the runner image moves to a new major version.